docs/en_us/platform_api/source/bookmarks/overview.rst

```diff
@@ -0,0 +1,30 @@
+.. _edX Bookmarks API Overview:
+ 
+ 
+################################################
+Bookmarks API Overview
+################################################
+ 
+You use the Bookmarks API to add a bookmark to an XBlock, get a bookmark or
+list of bookmarks for a course, or delete a bookmark. You can use the
```

Comment:
is it get a bookmark for a course, or get a specific user's bookmarks for a course?